petastorm/tests/test_pytorch_dataloader.py

```diff
@@ -72,6 +72,22 @@ def test_simple_read(synthetic_dataset, reader_factory):
         _check_simple_reader(loader, synthetic_dataset.data, BATCHABLE_FIELDS - {TestSchema.sensor_name})
 
 
+@pytest.mark.parametrize('reader_factory', ALL_READER_FLAVOR_FACTORIES)
+def test_simple_read_with_small_shuffle(synthetic_dataset, reader_factory):
+    with DataLoader(reader_factory(synthetic_dataset.url, schema_fields=BATCHABLE_FIELDS,
+                                   transform_spec=TransformSpec(_sensor_name_to_int)),
+                    shuffle_size=20) as loader:
```

Comment:
Can you please verify that the shuffling is actually happening?
This test implementation may help: https://github.com/uber/petastorm/blob/a7234bc8af30762676129099811643f9451f938a/petastorm/tests/test_end_to_end.py#L248

Reply:
Done, Thanks